From 5fcad0d86d985b068eb97e55b55f7be87cd1b008 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Wed, 12 May 2021 17:01:29 +0200
Subject: [PATCH] arm64: uaccess: include scheduler.h

Qualcomm SSDK exposed a bug in which the uaccess is
using a struct and header definition that are not
included.

So, simply: #include <linux/sched.h> to fix it.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 arch/arm64/include/asm/uaccess.h | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/include/asm/uaccess.h
+++ b/arch/arm64/include/asm/uaccess.h
@@ -24,6 +24,8 @@
 #include <asm/memory.h>
 #include <asm/extable.h>
 
+#include <linux/sched.h>
+
 #define get_fs()	(current_thread_info()->addr_limit)
 
 static inline void set_fs(mm_segment_t fs)
